cmd/thanos/receive.go

```diff
@@ -194,7 +194,7 @@ func runReceive(
 	if err != nil {
 		return err
 	}
-	dialOpts, err := extgrpc.StoreClientGRPCOpts(logger, reg, tracer, rwServerCert != "", rwClientCert, rwClientKey, rwClientServerCA, rwClientServerName)
+	dialOpts, err := extgrpc.StoreClientGRPCOpts(logger, nil, reg, tracer, rwServerCert != "", rwClientCert, rwClientKey, rwClientServerCA, rwClientServerName)
```

Comment:
unrelated to this PR (we should tackle this ): this rather lonely looking `nil` here (clientInstance) is a bit hard to reason about, maybe we could combine this in the effort done in #2267, cc @PhilipGough @kakkoyun ?

Reply:
It has of course now turned into a lonely looking `""`